code/modules/reagents/reagent_containers/syringes.dm

```diff
@@ -41,8 +41,8 @@
 	..()
 	update_icon()
 
-/obj/item/weapon/reagent_containers/syringe/attack_paw()
-	return attack_hand()
+/obj/item/weapon/reagent_containers/syringe/attack_paw(mob/user)

```

Comment:
I just object to each and every mob having its own "attack" proc. Not your fault.